ereolen_support/ereolen_support.admin.form.inc

```diff
@@ -142,6 +142,14 @@ function ereolen_support_config_form($form, &$form_state) {
     '#default_value' => variable_get('ereolen_support_submission_text'),
   ];
 
+  $form['texts']['ereolen_support_lead'] = [
+    '#type' => 'text_format',
+    '#title' => t('Lead text'),
+    '#description' => t('The text displayed above the form.'),
+    '#default_value' => variable_get('ereolen_support_lead')['value'],
```

Comment:
You should use `variable_get('ereolen_support_lead')['value'] ?? NULL` (or similar) to prevent warnings when the `ereolen_support_lead` variable is not set (on first edit of the support form settings) (cf. https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op).